client/state/notices/middleware.js

```diff
@@ -98,6 +157,14 @@ export const handlers = {
 	[ POST_RESTORE_FAILURE ]: onPostRestoreFailure,
 	[ POST_RESTORE_SUCCESS ]: dispatchSuccess( translate( 'Post successfully restored' ) ),
 	[ POST_SAVE_SUCCESS ]: onPostSaveSuccess,
+	[ PUBLICIZE_CONNECTION_CREATE ]: onPublicizeConnectionCreate,
+	[ PUBLICIZE_CONNECTION_CREATE_FAILURE ]: dispatchErrorMessage,
```

Comment:
Are these error messages actually translated? For most endpoints messages serve as additional debug info, but aren't meant to be displayed directly to the user.